services/oauth/v2/destination_info.go

```diff
@@ -16,12 +17,19 @@ type DestinationInfo struct {
 	DefinitionConfig map[string]interface{}
```

Comment:
For now add the exact type then we don't need nestedMap lookup

DefinitionConfig = struct {
    refreshOAuthToken bool
}